<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWidget</class>
 <widget class="QWidget" name="DebugWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1519</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_cams">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1471</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>最近拍摄照片</string>
   </property>
   <widget class="QGroupBox" name="groupBox_case1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>681</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Case1</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView_case1_FR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_case1_UB">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_case1_LD">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_FR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>FR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_UB">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>UB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_LD">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>200</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>LD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_case2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>681</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Case2</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView_case2_LD">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_case2_UB">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_LD_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>200</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>LD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_FR_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>FR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_case2_FR">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_UB_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>81</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>UB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>1471</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>机械爪操作</string>
   </property>
   <widget class="QGroupBox" name="groupBox_left">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>711</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>左爪</string>
    </property>
    <widget class="QGroupBox" name="groupBox_left_rotation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>321</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>旋转</string>
     </property>
     <widget class="QPushButton" name="btn_left_to90">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>至90°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_to180">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>至180°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_clockwise">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顺时针</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_anticlockwise">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>逆时针</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_clockwise_small">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顺时针(微调)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_anticlockwise_small">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>逆时针(微调)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_180">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>180°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_5circle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>连续五圈</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_left_open_close">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>321</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>开合</string>
     </property>
     <widget class="QPushButton" name="btn_left_open_small">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开(微调)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_open_all">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>全开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_open">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_close_half">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>半合</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_close">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>合上</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_left_close_small">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>合上(微调)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_right">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>701</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>右爪</string>
    </property>
    <widget class="QGroupBox" name="groupBox_right_rotation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>321</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>旋转</string>
     </property>
     <widget class="QPushButton" name="btn_right_to90">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>至90°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_to180">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>至180°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_clockwise">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顺时针</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_anticlockwise">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>逆时针</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_clockwise_small">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顺时针(微调)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_anticlockwise_small">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>逆时针(微调)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_180">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>180°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_5circle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>连续五圈</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_right_open_close">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>321</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>开合</string>
     </property>
     <widget class="QPushButton" name="btn_right_open_small">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开(微调)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_open_all">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>全开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_open">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_close_half">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>半合</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_close">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>合上</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_right_close_small">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>合上(微调)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
